include/belle.hh

```diff
@@ -1196,10 +1196,15 @@ private:
 
   private:
 
+#ifndef _MSC_VER
     // generic lambda for sending different types of responses
-    static auto const constexpr send = [](auto self, auto&& res) -> void
+    static auto constexpr send = [](auto self, auto&& res) -> void
+#else
+    template <typename SELF_T, typename RES_T>
```

Comment:
To follow the naming convention used in other templates, could the variables be named:
```cpp
template <typename Self, typename Res>
```